src/taskpane/server.js

```diff
@@ -535,8 +557,7 @@ function putUpdater(body, currentUser, PUTUrl) {
 // @param: projectId - int of the current project
 // @param: artifactId - int of the current artifact
 // @param: parentId - optional int of the relevant parent to attach the artifact too
-function postArtifactToSpira(entry, user, projectId, artifactTypeId, parentId) {
-
+function postArtifactToSpira(entry, user, projectId, artifactTypeId, parentId, isComment, isAssociation) {

```

Comment:
Does it make sense to have so many options in this one function? Could you split it into 3 functions?
postArtifactToSpira, postCommentToSpira, postAssociationToSpira

Reply:
The original idea here was to simplify - but it ended up complicating it. I created these 3 different functions, as suggested.